app/react/Attachments/components/UploadSupportingFile.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/prop-types */
+
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import { Translate } from 'app/I18N';
+import { Icon } from 'UI';
+
+import { uploadAttachment } from '../actions/actions';
+import AttachmentsModal from './AttachmentsModal';
+
+interface UploadSupportingFileProps {
+  entityId: string;
+  entitySharedId: string;
+  storeKey: string;
+  progress?: any;
+}
+
+const UploadSupportingFile: React.FC<UploadSupportingFileProps> = props => {
```

Comment:
If you specify the props type, don't need the eslint suppression comment
`props: UploadSupportingFileProps`

Reply:
No need for prop types if we are using typescript.

Reply:
@konzz can you apply here the same changes for the prop types?